rvv-intrinsic-generator/Makefile.api

```diff
@@ -14,8 +14,8 @@
 # limitations under the License.
 ###############################################################################
 
-CFLAGS?=-O -Werror=implicit-function-declaration
-EXTRA_CFLAGS?=-march=rv64gcv_zvfh -menable-experimental-extensions
+CFLAGS?=-O -Werror=implicit-function-declaration -Wno-incompatible-pointer-types
```

Comment:
Wait, why `-Wno-incompatible-pointer-types`?

Reply:
The test script originally treats some warning with notes as errors. I just modified the script to expose an option to treat warning as error by parsing the last line of the log and remove this addition of `-Wno-incompatible-pointer-types`.